client/templates/main.js

```diff
@@ -9,14 +9,14 @@ if (Meteor.isClient) {
     }
 
     const defaultLang = 'en'
-    const urlLang = FlowRouter.getQueryParam('lang')
+    const localStorageLang = localStorage.getItem('lang');
```

Comment:
Can we call the localstorage item also languageCode to stay in sync with the rest? :)